spec/features/02_search_spec.rb

```diff
@@ -0,0 +1,29 @@
+require_relative '../spec_helper'
+
+RSpec.describe 'Search functionality', type: :feature do
+  before(:each) do
+    visit('/')
+  end
+
+  it 'Verifies product search without options' do
+    $search.search_product('Xiaomi mi play')
+    expect(page).to have_content(ProductsConstants::XIAOMI)
+  end
+
+  it 'Verifies search without options' do
+    $search.search_product('garmin 235')
+    expect(page).to have_content('Garmin Forerunner 235 Black/Grey (010-03717-55)')
+  end
```

Comment:
what is the difference between these two scenarios?

Reply:
This temporary test was added to verify quantity in Test Results. Also, It has been removed.